### ISSUE TITLE
P2P default ports conflict with Aleph-VM

### ISSUE THREAD

**Issue by hoh** [aleph-im/pyaleph#168]
The P2P daemon `control_port` and `listen_port` are configured to use the same ports as Aleph-VM Connector and Supervisor.

```json
            "control_port": 4020,
            "listen_port": 4021,
```

## Impact

This means that users would have to tweak the configuration in order to run Aleph-VM and PyAleph on the same system. While this is not meant to happen in production, it may happen in tests.

## References

https://github.com/aleph-im/pyaleph/blob/dev/src/aleph/config.py#L16
https://github.com/aleph-im/pyaleph/blob/dev/deployment/docker-compose/docker-compose.yml#L43

On Aleph-VM:
https://github.com/aleph-im/aleph-vm/blob/main/vm_supervisor/conf.py#L62
https://github.com/aleph-im/aleph-vm/blob/main/vm_supervisor/conf.py#L90
And the documentation to launch the connector via Docker.

**Comment by odesenfans** [comment]
It's easy to change, no worries. Modifying it.

**Comment by odesenfans** [comment]
Resolved by #170.
